src/libraries/System.Security.Claims/src/System/Security/Claims/ClaimsIdentity.cs

```diff
@@ -946,7 +946,7 @@ internal string DebuggerToString()
                 claimsCount++;
             }
 
-            return $"Identity Name = {Name ?? "(null)"}, IsAuthenticated = {IsAuthenticated}, Claims Count = {claimsCount}";
+            return $"Identity Name = {Name ?? "(null)"}, IsAuthenticated = {IsAuthenticated ? "true" : "false"}, Claims Count = {claimsCount}";
```

Comment:
I wish the compiler was smart enough to handle it, as I always run into this problem myself.

> Error	CS8361	A conditional expression cannot be used directly in a string interpolation because the ':' ends the interpolation. Parenthesize the conditional expression.	

```suggestion
            return $"Identity Name = {Name ?? "(null)"}, IsAuthenticated = {(IsAuthenticated ? "true" : "false")}, Claims Count = {claimsCount}";
```